src/components/DataElement/CitiesSplit.jsx

```diff
@@ -20,19 +23,31 @@ export default function CitiesSplit({ isLoading, data }) {
       >
         {isLoading && <Loader />}
         {data &&
-          <ResponsiveContainer height={data.length * 32} width={'99%'}>
-            <BarChart
-              data={data}
-              layout="vertical"
-            >
-              <YAxis dataKey="city" type="category" tick={{ fill: theme.colors.contentPrimary }} width={100} />
-              <XAxis type="number" hide />
-              <Tooltip
-                formatter={value => [value, 'Liczba']}
-              />
-              <Bar dataKey="count" fill={theme.colors.accent} />
-            </BarChart>
-          </ResponsiveContainer>}
+          <DataContext.Consumer>{({ setPosition, setZoom }) => (
```

Comment:
Works well, but there are some improvements I would implement. First of all, in DataContext I would store just city name. Then basing on that I would find its location from `cities` array and change the map's position. I wouldn't also use `DataContext.Consumer` when you can use `useData` hook above.

Reply:
Done